src/main/java/uk/gov/hmcts/darts/common/helper/SystemUserHelper.java

```diff
@@ -20,8 +24,18 @@ public class SystemUserHelper {
 
     private Map<String, String> systemUserGuidMap;
 
+    private final UserAccountRepository userAccountRepository;
+
     public String findSystemUserGuid(String systemUserName) {
         return systemUserGuidMap.get(systemUserName);
     }
 
+    public UserAccountEntity getSystemUser() {
```

Comment:
should this not be called `getHousekeepingUser`?

Reply:
ACCEPT